i18n/sources.txt

```diff
@@ -0,0 +1,5 @@
+lib/config/presets/internal/default.ts
+lib/modules/platform/github/index.ts
+lib/workers/repository/onboarding/pr/config-description.ts
+lib/workers/repository/onboarding/pr/index.ts
+lib/workers/repository/onboarding/pr/pr-list.ts
```

Comment:
Eventually nearly every file needs to be added to this list?

Reply:
Maybe, but we can leverage this command to write this file.

```shell
make i18n/sources.txt
```